sklearn/inspection/tests/test_partial_dependence.py

```diff
@@ -836,3 +836,55 @@ def test_kind_average_and_average_of_individual(Estimator, data):
     pdp_ind = partial_dependence(est, X=X, features=[1, 2], kind="individual")
     avg_ind = np.mean(pdp_ind["individual"], axis=1)
     assert_allclose(avg_ind, pdp_avg["average"])
+
+
+def test_grid_from_X_missing_values_numerical_features():
+    """Check the behavior when X contains missing values with numerical features.
+
+    When generating the grid, we should ignore the missing values to avoid introducing
+    a bias in the partial dependence computation. Therefore, we make sure that there is
+    no missing values in the grid.
+
+    Non-regression test for:
+    https://github.com/scikit-learn/scikit-learn/issues/25401
```

Comment:
```suggestion
    #25401
```
Though there seem to be examples of both styles 🤷

Reply:
I will always put a link because I can click and be redirected from the IDE.

Reply:
but when reviewing I intend to no bother people on this one ;)

Reply:
I liked that I could click, but then thought "Guillaume always comments to make it an issue number in a specific format in my PRs". Let's leave it as a link :D

Reply:
True that if I see `#25401` I would ask for `gh-25401` to make it explicit that this is linked to GitHub :)

Reply:
I also like the full link such that I can click on it or just c&p in the browser. But in the end, I'm OK with all versions as long as it is clear.